scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -5,18 +5,19 @@ def printBigHeader( author = 'Jungle Cat Software', package = 'AWS-Toolkit', fun
   firstLineLen = len( firstLine )
   secondLine = function
   secondLineLen = len( secondLine )
-
+  
   longestLineLen = firstLineLen if firstLineLen >= secondLineLen else secondLineLen
   linePadLen = longestLineLen + 2
   headerLineLen = linePadLen + 2
 
-  print( ( '{0:*^' + str( headerLineLen ) + '}' ).format( '' ) )
-  print( ( '*{0: ^' + str( linePadLen ) + '}*' ).format( firstLine ) )
-  print( ( '*{0: ^' + str( linePadLen ) + '}*' ).format( secondLine ) )
-  print( ( '{0:*^' + str( headerLineLen ) + '}' ).format( '' ) )
+  print('*' * headerLineLen)
```

Comment:
I don't know why I didn't do this one, I guess I just got excited for the shiny new hammer you pointed out to me...